### ISSUE TITLE
Add `aria` prop to`Tabs` and `Tabs Bar` component

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#4481]
### Scope

Currently the `Tabs` and `Tabs Bar` component are missing te possibility to provide a meaningful name and description to the tabs.

- Add `aria-label`
- Add `aria-description`

### Acceptance criteria

- [x] Tabs and Tabs Bar should have a possibility to provide accessible name and description
